app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -92,6 +92,36 @@
   }
 }
 
+.onboarding-content-separator {
```

Comment:
Note for reviewers:

I have added these CSS classes, as I was struggling a bit to add the classes (crayons/utility) directly to html a bit.
If you have any suggestions to optimise this feel free to add. I was mainly getting stuck on background/background-color.

Other than that I have added all possible utility css classes directly in html code.